config/base/derive/src/view.rs

```diff
@@ -83,6 +83,7 @@ mod gen {
                 }
             });
         remove_attr_struct(&mut ast, "view");
+        remove_attr_struct(&mut ast, "config");
```

Comment:
why do we need to clear `config` attribute here?

Reply:
We don't actually need to, as the program still compiles with it present, but I guess it doesn't make sense to have `config` on `view`s. Due to this attribute is in charge of documentation loading and querying, we don't need it on views currently as none of those actions are being performed on them. Of course this might change later. I remember a [comment](https://github.com/hyperledger/iroha/issues/2118#issuecomment-1181471140) you made some time ago how `Combine` trait could help loading config on the fly, but enabling that functionality will require a separate line of thinking. I guess I'll create an issue to keep track of this.